obligation-cordapp/build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext.corda_release_group = 'net.corda'
-    ext.corda_release_version = 'corda-3.0'
-    ext.corda_gradle_plugins_version = '3.0.9'
+    ext.corda_release_version = '3.3-corda'
+    ext.corda_gradle_plugins_version = '3.1.0'
```

Comment:
Actually I've just looked here: https://github.com/corda/corda/blob/release-V3/constants.properties and it appears the V3 branch is on 3.2.1 of the plugins. Would you mind updating it and checking that works?

Reply:
I've updated and it seems to be fine